### PR TITLE
Add tool dependency definition for hisat2.

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

File: packages/package_hisat_2_0/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  we designed and implemented a graph FM index (GFM), an original approach and its first
+  implementation to the best of our knowledge. In addition to using one global GFM index that
+  represents the general population, HISAT2 uses a large set of small GFM indexes that
+  collectively cover the whole genome (each index representing a genomic region of 56 Kbp,
+  with 55,000 indexes needed to cover the human population). These small indexes (called
+  local indexes), combined with several alignment strategies, enable rapid and accurate
+  alignment of sequencing reads. This new indexing scheme is called a Hierarchical Graph
+  FM index (HGFM).'
+homepage_url: https://github.com/galaxyproject/tools-iuc/
+remote_repository_url: https://github.com/galaxyproject/tools-iuc/blob/master/packages/package_hisat_2_0/tool_dependencies.xml
```
Comment (nsoranzo) [inline packages/package_hisat_2_0/.shed.yml]: I think this should be the directory, i.e.: `https://github.com/galaxyproject/tools-iuc/blob/master/packages/package_hisat_2_0/`

Comment (martenson) [inline packages/package_hisat_2_0/.shed.yml]: agree, either directory with .shed.yml or .shed.yml itself

Comment (nsoranzo) [inline packages/package_hisat_2_0/.shed.yml]: Directory is more consistent with tools, for which there may be many files, not just 1.

**Comment by nsoranzo** [comment]
LGTM.

**Comment by bgruening** [comment]
:+1:

**Pull request merged**